libbeat/scripts/kibana_export.py

```diff
@@ -0,0 +1,47 @@
+from elasticsearch import Elasticsearch
+import argparse
+import os
+import json
+
+
+def dump_objects(es, output_directory, doc_type):
+    res = es.search(
+        index='.kibana',
```

Comment:
Agree, for development we don't need to configure the Kibana index. It would be nice to add in the docs that the latest Kibana is requested for importing/exporting the dashboards.